TeamMemberFiles/katie.oconnell.txt

```diff
@@ -0,0 +1,5 @@
+Katie O'Connell
+Please call me Katie
+I am in 294
+I am most familiar with Java at this time, but ready to learn more.
+I am looking forward to starting this project!
```

Comment:
Awesome work Katie  -
Please make sure to clean up your repo so that no private information is exposed.